lib/galaxy/tools/__init__.py

```diff
@@ -2352,8 +2347,8 @@ def to_json (self, trans, kwd={}, is_workflow=False):
         tool_message = ''
         if job:
             try:
-                job_params = job.get_param_values( trans.app, ignore_errors = True )
-                job_messages = self.check_and_update_param_values( job_params, trans, update_values=False )
+                job_params = job.get_param_values( trans.app, ignore_errors=True )
+                self.check_and_update_param_values( job_params, trans, update_values=False )
```

Comment:
Instead of removing the unused variable, it looks like the right fix here might be to use the info messages returned by check_and_update_param_values.